test/fixImageSize.js

```diff
@@ -1,23 +1,14 @@
-const sharp = require('sharp');
-const sizeOf = require('image-size');
-const fs = require('fs');
- 
-function getFileSizeMB(fileStats){
-    return (fileStats.size) / (1024*1024);
-}
+import {imageSizeChecker} from './library.js';
+import {getAllFiles} from './library.js';
+import {fixImage} from './library.js';
```

Comment:
We can destructure all of these at once:

```suggestion
import { fixImage, getAllFiles, imageSizeChecker} from './library.js';
```